pylab/website/templates/website/about.html

```diff
@@ -0,0 +1,63 @@
+{% extends "base.html" %}
+{% load trans blocktrans from i18n %}
+
+
+{% block content %}
+  <div class="about">
+    <h1>{% trans "About Summer Python Workshop" %}</h1>
+
+    <p>{% blocktrans %}During this event everybody can learn how to program in
+    Python and Django by creating <a href="http://pylab.lt">pylab.lt</a>
+    website.{% endblocktrans %}</p>
+
+    <p>{% blocktrans %}Every workshop participant has to choose individual task,
+    which he would be able to complete in one week. Two hours long meeting is
+    held each week, during which participants present how they managed to complete
+    their tasks and what issues they faced. Advanced programmers give advices and
+    help to solve problems during this meeting. Participants can choose next tasks
+    if they managed to finish their task successfully.{% endblocktrans %}</p>
+
+    <p>{% blocktrans %}"Summer Python workshop" meetings are held every Monday
+    from 18:00 till 20:00 at Čiurlionio g. 13-1.{% endblocktrans %}</p>
+
+    <iframe width="830" height="350" frameborder="0" scrolling="no" marginheight="0" marginwidth="0"
+    src="http://www.openstreetmap.org/export/embed.html?bbox=25.26177942752838%2C54.68205628977621%2C25.264630615711212%2C54.68296184199744&amp;layer=mapnik&amp;marker=54.6825090684119%2C25.263205021619797"
+    style="border: 1px solid black"></iframe>
+
+    <h1>{% trans "Python workshops policy" %}</h1>
+
+    <p>{% blocktrans %}Intellectual property and source code created during Python
+    workshops by participants belong to Python workshops.{% endblocktrans %}</p>
+
+    <p>{% blocktrans %}Python workshop's strategical and financial solutions are
+    made democratically: Python workshops participants vote and majority's will is
+    executed.{% endblocktrans %}<p>
+
+    <p>{% trans "Each Python workshops participant has to:" %}
+    <ul>
+      <li>{% trans "Accept open source and open knowledge ideologies." %}</li>
+      <li>{% blocktrans %}Respect and protect human rights and liberties.
+          Especially right for pricacy.{% endblocktrans %}</li>
+      <li>{% trans "Respect Lithuania and foster its prosperity." %}</li>
```

Comment:
Didn't know how to put it, that prorussians or other kind of destructive forces are not proper for these workshops. I think we have to explicitly state, that we do not create tools, which would harm Lithuania's suverenitet. On the contrary we create transparency tools.